js_modules/dagit/packages/core/src/launchpad/OpSelector.tsx

```diff
@@ -45,25 +51,41 @@ const SOLID_SELECTOR_QUERY = gql`
       }
     }
   }
-  ${OP_NODE_INVOCATION_FRAGMENT}
+  ${GRAPH_EXPLORER_SOLID_HANDLE_FRAGMENT}
 `;
 
 export const OpSelector = (props: IOpSelectorProps) => {
-  const {serverProvidedSubsetError, onChange, pipelineName, repoAddress} = props;
+  const {
+    serverProvidedSubsetError,
+    onChange,
+    pipelineName,
+    repoAddress,
+    onFlattenGraphsChange,
+  } = props;
   const [focused, setFocused] = React.useState(false);
   const inputRef = React.useRef<HTMLInputElement>(null);
 
+  const flattenGraphs = props.flattenGraphs || false;
   const selector = {...repoAddressToSelector(repoAddress), pipelineName};
   const repo = useRepository(repoAddress);
   const isJob = isThisThingAJob(repo, pipelineName);
-
   const {data, loading} = useQuery<OpSelectorQuery>(SOLID_SELECTOR_QUERY, {
-    variables: {selector},
+    variables: {selector: selector, requestScopeHandleID: flattenGraphs ? undefined : ''},
```

Comment:
It's a bit strange that these result in different behaviors, but I don't have a good solution in mind offhand.

Reply:
i think this could be a backend issue/fact that `""` means the parent handle is empty and `None` would return all handles recursively.